src/sample/classes/StudentController.java

```diff
@@ -18,67 +21,67 @@
 import javafx.stage.Stage;
 
-public class StudentController implements Initializable {
-  @FXML
-  private Label label;
-  @FXML
-  private AnchorPane anchorid;
-  @FXML
-  private TextField textfield;
+public class StudentController implements Initializable{
+@FXML
+private Label label;
+@FXML
+private AnchorPane anchorid;
+@FXML
+private TextField textfield;
 
-  @FXML
-  public void handleButtonAction(ActionEvent event) throws IOException {
+@FXML
+public void handleButtonAction(ActionEvent event) throws IOException{
     final DirectoryChooser dirchooser = new DirectoryChooser();
 
     Stage stage = (Stage) anchorid.getScene().getWindow();
 
     File file = dirchooser.showDialog(stage);
 
-    // input the text file for the course list
-    // try {
-    // File fileObj = new File("student1.txt");
-    // Scanner fileReader = new Scanner(fileObj);
-    // while (fileReader.hasNextLine()) {
-    // registeredCourse.getItems().list1.add(fileReader.nextLine());
-    // }
-    // fileReader.close();
-    // } catch (FileNotFoundException e) {
-    // System.out.println("An error occurred.");
-    // e.printStackTrace();
-    // }
-    if (file != null) {
-      List list1 = new List();
-      list1.add("Java");
-      list1.add("Android");
-      list1.add("Kotlin");
-      list1.add("Android");
-
-      System.out.println("Path: " + file.getAbsolutePath());
-      textfield.setText(file.getAbsolutePath());
-
-      PdfWriter pdfWriter = new PdfWriter(file.getAbsolutePath() + "/CourseList.pdf");
-
-      PdfDocument pdfDocument = new PdfDocument(pdfWriter);
-      pdfDocument.addNewPage();
-
-      Document document = new Document(pdfDocument);
-      document.add(list1);
-
-      document.close();
-
+    if(file !=null)
+    {   
+        List list1= new List();
+      
+        //input the text file for the course list
+        try {
+            File fileObj = new File("student1registeredCourse.txt");
+             Scanner fileReader = new Scanner(fileObj);
+             while (fileReader.hasNextLine()) {
+               
+            list1.add(fileReader.nextLine());
+             }
+             fileReader.close();
+           } catch (FileNotFoundException e) {
+             System.out.println("An error occurred.");
+             e.printStackTrace();
+           }
+
+           
+        System.out.println("Path: " + file.getAbsolutePath());
+        textfield.setText(file.getAbsolutePath());
+        
+        
+        PdfWriter pdfWriter = new PdfWriter(file.getAbsolutePath() + "/CourseList.pdf");
+
+        PdfDocument pdfDocument = new PdfDocument(pdfWriter);
+        pdfDocument.addNewPage();
+
+        Document document = new Document(pdfDocument);
+        document.add(list1);
+
+ 
+        document.close();
+        stage.close();
     }
 
-  }
 
-  @FXML
-  public void confirmbutton(ActionEvent event) {
-    Stage stage = (Stage) anchorid.getScene().getWindow();
-    stage.close();
+  
+}
+
```

Comment:
when the user click confirm can it show the `The pdf is created`, then ask the user to close the window?

Reply:
i removed the confirm button as i think it's redundant to let users click twice just like when we upload files to Gdrive, it won't ask us to confirm again. 

For the 'The pdf is created' message I can try